def myfunc():
    return 1
print(myfunc())
#known error for testing lint